osbs/cli/main.py

```diff
@@ -49,6 +49,7 @@ def _print_pipeline_run_logs(pipeline_run, user_warnings_store):
     except Exception as ex:
         logger.error("Error during fetching logs for pipeline run %s: %s",
                      pipeline_run_name, repr(ex))
+        return 1
```

Comment:
This seems very unpythonic to me, I would maybe prefer raising some exception here
and then caling this method in a try block

Reply:
Personally, I don't like to see:
-1 Completely different types being returned
-2 Multiple exit points. Inverting the logic of the if statement, assigning a return value, and exiting once with a return of that value is much cleaner and easier to follow

Reply:
we are silencing exception here so we don't have to catch it later, so why again raise another?

Reply:
then don't silence it and just raise it, so outside code depending on this function knows there was a failure based on an exception and not return value of 1.

Reply:
BUT following functions still have to print the summary ......
I am only returning 1 so I have some indications that get_logs really failed and if build still didn't finish additionally cancel it

Reply:
I've added return 1 also to the case when it isn't iterable

Reply:
ok returning 1 if everything is fine so now I am returning just int

Reply:
Ack, could we have a docstring that explains what the return value means then?

Reply:
added